client/src/react-app-env.d.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference types="react-scripts" />
+
+// https://github.com/DefinitelyTyped/DefinitelyTyped/issues/35707#issue-447896832
+declare module 'babel-plugin-relay/macro';
+// {
+// 	export { graphql } from 'react-relay'
+// }
```

Comment:
> Not sure why import {graphql} from 'react-relay'; does not work for you. I can help investigate if you want.

For archival purposes:

The answer is that using `babel-plugin-relay/macro` and `babel-plugin-macros` circumvents our need to add a Babel plugin (`babel-plugin-relay`) to the build process, which we cannot do using Create React App out-of-the-box. 

CRA uses the opaque `react-scripts` for its build process to abstract it away and make it zero-config and so "easy" to use, but it will ignore any `.babelrc` or webpack config, etc. that you try to add to your project.

https://relay.dev/docs/en/installation-and-setup#set-up-babel-plugin-relay